static/nodejs_server/query.ts

```diff
@@ -248,13 +286,10 @@ export async function getQueryResult(
   // Get the nl detect-and-fulfill result for the query
   let nlResp = null;
   try {
-    nlResp = await axios
-      // Set "mode=strict" to use heuristic detector, disable using default place,
-      // use a higher SV threshold and avoid multi-verb queries
-      .post(
-        `${apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=${mode}&client=${client}`,
-        {}
-      );
+    nlResp = await axios.post(
+      `${apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=${mode}&client=${client}&detector=${QUERY_DETECTOR}&varThreshold=${varThreshold}`,
```

Comment:
TODO: do we want to control related-thing generation?

Reply:
added todo